### DISCUSSION TITLE
How can I update the electronic temperature (Te) to align with the nuclear temperature (Tn) during quenching?

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/deepmd-kit#3067] [Q&A]
I have developed a deep potential model dependent on electron temperature (TD-DP) and aim to apply it to study the quenching process of liquid silicon from 2000 K to 300 K. I've observed that Te is set as a fixed input parameter in "pair_style deepmd graph.pb fparam 2000". 

In the process of quenching Tn, how can I dynamically adjust Te accordingly?

**Comment by njzjz** [comment]
There are several available keys `fparam_from_compute`, `aparam_from_compute`, and `ttm`. You can find details from [documentation](https://docs.deepmodeling.com/projects/deepmd/en/master/third-party/lammps-command.html#pair-style-deepmd).
